bazel/ci/golicenses.sh.in

```diff
@@ -27,65 +27,73 @@ not_allowed() {
   err=1
 }
 
-${go} mod download
+license_report() {
+  PATH="$(dirname "${go}"):${PATH}" \
+  GOROOT=$(${go} env GOROOT) \
+  GOPATH=$(${go} env GOPATH) \
+  GOCACHE=$(${go} env GOCACHE) \
+    ${golicenses} report ./... | {
+    while read -r line; do
 
-err=0
+      pkg=${line%%,*}
+      lic=${line##*,}
 
-PATH="$(dirname "${go}"):${PATH}" \
-GOROOT=$(${go} env GOROOT) \
-GOPATH=$(${go} env GOPATH) \
-GOCACHE=$(${go} env GOCACHE) \
-  ${golicenses} csv ./... | {
-  while read -r line; do
+      case ${lic} in
+      Apache-2.0 | BSD-2-Clause | BSD-3-Clause | ISC | MIT) ;;
 
-    pkg=${line%%,*}
-    lic=${line##*,}
+      MPL-2.0)
+        case ${pkg} in
+        github.com/siderolabs/talos/pkg/machinery/config/encoder) ;;
 
-    case ${lic} in
-    Apache-2.0 | BSD-2-Clause | BSD-3-Clause | ISC | MIT) ;;
+        github.com/letsencrypt/boulder) ;;
 
-    MPL-2.0)
-      case ${pkg} in
-      github.com/siderolabs/talos/pkg/machinery/config/encoder) ;;
+        github.com/hashicorp/*) ;;
 
-      github.com/letsencrypt/boulder) ;;
+        *)
+          not_allowed
+          ;;
+        esac
+        ;;
 
-      github.com/hashicorp/*) ;;
+      AGPL-3.0)
+        case ${pkg} in
+        github.com/edgelesssys/constellation/v2) ;;
 
-      *)
-        not_allowed
+        github.com/edgelesssys/constellation/v2/operators/constellation-node-operator/v2/api/v1alpha1) ;;
+
+        *)
+          not_allowed
+          ;;
+        esac
         ;;
-      esac
-      ;;
 
-    AGPL-3.0)
-      case ${pkg} in
-      github.com/edgelesssys/constellation/v2) ;;
+      Unknown)
+        case ${pkg} in
+        github.com/edgelesssys/go-tdx-qpl/*) ;;
 
-      github.com/edgelesssys/constellation/v2/operators/constellation-node-operator/v2/api/v1alpha1) ;;
+        *)
+          not_allowed
+          ;;
+        esac
+        ;;
 
       *)
-        not_allowed
+        echo "unknown license: ${line}"
+        err=1
         ;;
       esac
-      ;;
 
-    Unknown)
-      case ${pkg} in
-      github.com/edgelesssys/go-tdx-qpl/*) ;;
+    done
+    exit "${err}"
+  }
+}
 
-      *)
-        not_allowed
-        ;;
-      esac
-      ;;
+${go} mod download
 
-    *)
-      echo "unknown license: ${line}"
-      err=1
-      ;;
-    esac
+err=0
 
-  done
-  exit "${err}"
-}
+if test -v silent; then
```

Comment:
```suggestion
if test -v SILENT; then
```

by convention, any environment variables that come from the outside (are not bash variables set locally in the script) should use all caps: https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names